libraries/botbuilder-ai/package.json

```diff
@@ -33,6 +33,8 @@
     "adaptive-expressions": "4.1.6",
     "botbuilder-core": "4.1.6",
     "botbuilder-dialogs": "4.1.6",
+    "botbuilder-dialogs-declarative": "4.1.6",
+    "botbuilder-runtime-core": "4.1.6",
```

Comment:
`LuisComponentRegistration` which had dependencies a GA package; while `LuisBotComponent` depends on two preview packages.

Let's sync offline on our messaging to customers and make to mention in the release notes that we've hard deprecated `LuisComponentRegistration`.

Reply:
That's an oversight on my part. I can shim the new `BotComponent` back into the `ComponentRegistration` style. I missed that `botbuilder-ai` wasn't still in `preview`.

Reply:
Perfect, thank you!